samza-test/src/main/java/org/apache/samza/test/framework/system/descriptors/InMemoryOutputDescriptor.java

```diff
@@ -45,7 +45,7 @@
   }
 
   public InMemoryOutputDescriptor withPhysicalName(String physicalName) {
-    withPhysicalName(physicalName);
+    super.withPhysicalName(physicalName);
```

Comment:
Please correct me If i'm wrong here. Before this change, was this method going into a indefinite recursive call?

Reply:
yes if this is used, I mentioned this in a comment above